lib/quantizeAttributes.js

```diff
@@ -0,0 +1,348 @@
+'use strict';
```

Comment:
Did you check code coverage for this file?  See https://coveralls.io/github/AnalyticalGraphicsInc/gltf-pipeline